x-pack/plugins/search_indices/server/lib/status.ts

```diff
@@ -46,14 +46,14 @@ export async function fetchUserStartPrivileges(
       index: [
         {
           names: [indexName],
-          privileges: ['create_index'],
+          privileges: ['manage', 'delete'],
```

Comment:
getting user access privileges from `search_indices` plugin

Reply:
posting this here since I can't comment on an unchanged line - why is there a default value for the indexName? Doesn't seem useful in production.

Reply:
at least for `create_index` I need a name for that value to come back. `'manage'` may be the better check that doesn't require a specific index name.

Reply:
We need a list index names for [has privileges](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-has-privileges.html) api to work. The Index name in the list cannot be null or undefined. 
 I have modified to use [generated index name ](https://github.com/elastic/kibana/blob/main/x-pack/plugins/search_indices/public/components/start/elasticsearch_start.tsx#L38)that we create in the global empty state.

Reply:
@saarikabhasi This is outside the scope of the current pr but supplying a fake index name to check privileges sounds like a poor hack or broken api - either the index name is needed and the correct one should be supplied OR it shouldn't be needed at all and a request should be put into the ES team to change the api.

Reply:
commit : https://github.com/elastic/kibana/pull/197869/commits/b4d7cc6c6f6175da114da7c2f53c762af10c8bbd